docs/developers_guide/copyright_license.md

```diff
@@ -1,46 +1,206 @@
+
+
+
+## Frequently asked questions
+
+__Can RST be released with a different license?__<br/>
+To release RST with a different license, we would need permission from all of the copyright holders. This means that, in practice, it would be very difficult to re-license the software. This is by design of the GPL license--it ensures that the software can always be used freely. 
+
+
+__Can JHUAPL revoke the GPL license and take back control of RST?__<br/>
+No. JHUAPL gave permission for the RST to be re-licensed under the GPL in ~2012 (RST3.5), and they cannot revoke this permission. JHUAPL still owns the copyright to their own code, and RST versions 3.4 and earlier are still licensed by JHUAPL (not under GPL).
+
+
+__What happens if I don't include copyright information in my code?__<br/>
+In most countries, authors automatically hold the copyright to their own work even if they don't add a copyright notice. This is to protect the rights of people who are not aware of the law. However, omitting copyright information means that:
+
+- You may be breaching your employment contract
+- You are not complying with the GPL requirement to include copyright information
+- You may not get credit for your work
+- It is more difficult for you to prove that you are the copyright holder
+
+
+__Do I have to include my institution in the copyright line?__<br/>
+That depends on the terms of your employment. Check your employment contract or ask the research office. If you contribute to the RST in your free time, then just put your own name in the copyright line.
+
+__Can I copyright code to a generic "SuperDARN" organization?__<br/>
+No, the copyright holder has to be a legal entity or a person. Your employer may also object to this. If you contribute to the RST in your free time, then just put your own name in the copyright line.
+
+__Why does every RST source file have license notices at the top? Isn't it sufficient to include the license file in the top-level directory of RST?__<br/>
+Since the GPL allows users to modify and redistribute portions of the RST, it is possible that individual source files might become separated from the license file. If this happens, it will be unclear to users what their legal rights are to use/modify/distribute that version of the code (which is a violation of the GPL). Therefore, all source files should clearly indicate that they are licensed under the GPL (writing "see license.txt" is not sufficient).
+
+
+__If I add a new library to the RST, can that library have a different license?__<br/>
+No, the GPL requires that the whole "work" be released with the same license.
+
+
+
+__What are the licensing consequences of copying code from the RST into my own software project?__
```

Comment:
"consequences" means an effect the list suggests more what you have to do if you copy the code? Not sure if there is a better more concise word for it.

Reply:
Good point! Is this better?
```suggestion
__Am I allowed to copy code from the RST into my own software project? What do I have to do to comply with RST's license?__<br/>
```